common/src/main/java/com/mx/common/connect/Response.java

```diff
@@ -102,6 +103,10 @@ public final <T> T getBodyAs(Class<T> asClass) {
     return GSON.fromJson(this.body, asClass);
   }
 
+  public final <T> T getBodyAs(Type asType) {
+    return GSON.fromJson(this.body, asType);
+  }
+
```

Comment:
We are removing support for JSON bodies. You can still do this by providing a process block with `withProcessor` (to deserialize) and then call `getObject()` to retrieve the deserialized object.